tests/bdd/common/operations.py

```diff
@@ -0,0 +1,13 @@
+from common.apiclient import ApiClient
+
+
+class Pool(object):
+    @staticmethod
+    def __pools_api():
+        return ApiClient.pools_api()
+
+    # Delete all the pools
+    @staticmethod
+    def delete_all():
+        for pool in ApiClient.pools_api().get_pools():
+            ApiClient.pools_api().del_pool(pool.id)
```

Comment:
I think you mean to call the `__pools_api()` function here.

Reply:
dohh! It should be that indeed